build.gradle

```diff
@@ -43,57 +36,6 @@ repositories
     maven { url "http://repository.cloudera.com/cloudera/cloudera-repos/" }
 }
 
-idea {
-    project {
-        languageLevel = '1.8'
-    }
-}
-
-spotless {
```

Comment:
Why are we getting rid of spotless? This is very useful

Reply:
I tried re-organizing the build to have a smaller default build file. Now all the configuration for spotless, checkstyle, jacoco, sonar and tests go into `gradle/quality.gradle` and is imported by the main build file.

Reply:
See [this diff](https://github.com/osmlab/atlas-checks/pull/59/files#diff-c197962302397baf3a4cc36463dce5eaR19)!